packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -323,6 +338,24 @@ class SliderThemeData extends Diagnosticable {
   /// The color given to the [valueIndicatorShape] to draw itself with.
   final Color valueIndicatorColor;
 
+  /// The shape and behavior that will be used to draw the [Slider]'s track.
+  ///
+  /// This can be customized by implementing a subclass of
+  /// [SliderTrackShape].
+  final SliderTrackShape trackShape;
+
+  /// The shape and behavior that will be used to draw the [Slider]'s ticks.
+  ///
+  /// This can be customized by implementing a subclass of
+  /// [SliderTickShape].
```

Comment:
> [same as track shape]
> 
> This isn't really helpful (if it is helpful then the reader is out of their depth).
> 
> It might be worth pointing out how he tickShape's preferred size contribute's to the slider's overall size.
 Done
> It would be worth pointing out that the default value of tickShape is RoundSliderTrackShape.
Done 
> Should point out how this shape relates to trackHeight and the fooTrackColor properties.
Only the default RoundSliderTickShape uses the track height in its painting, its not a requirement for ticks in general. Should the trackHeight (and other details about default shape painting) comments focused within the default shape classes themselves?

Reply:
Yes

Reply:
Done